### PR TITLE
Bug 1741147: Add units to top consumers cpu and storage results.

### PR DESCRIPTION
**Pull request by rawagner**
Change storage description to 'by Average I/O time'

https://bugzilla.redhat.com/show_bug.cgi?id=1741147
https://bugzilla.redhat.com/show_bug.cgi?id=1741149

### CODE REVIEW

File: frontend/public/components/dashboard/top-consumers-card/strings.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export const CPU_DESC = 'CPU time';
 export const MEMORY_DESC = 'Memory consumption';
-export const STORAGE_DESC = 'Filesystem IO time';
+export const STORAGE_DESC = 'Average I/O time';
```
Comment (knowncitizen) [inline frontend/public/components/dashboard/top-consumers-card/strings.js]: Should this be Average Filesystem I/O time? I mean I guess that's implied.

Comment (rawagner) [reply frontend/public/components/dashboard/top-consumers-card/strings.js]: IMO `Average I/O time` is enough as you need to pick ` By Storage` from dropdowns above the metric first.

**Pull request merged**